client/checkout/classic/test/payment-processing.test.js

```diff
@@ -516,11 +561,11 @@ describe( 'Setup intent creation and confirmation', () => {
 		};
 
 		await createAndConfirmSetupIntent(
-			mockJqueryForm,
```

Comment:
I'm not sure how this was working before my changes - the signature of `createAndConfirmSetupIntent` is not `$form, intent, api`, but it is `intent, $form, api` 🤷

Reply:
I guess because `setupIntent()` still would've been called, just with a `$form` instead of an ID and we're only checking that it was called 🤷.